src/main/java/core/basesyntax/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.dao.OperationDaoImpl;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final String FILE_NAME
+            = "src/main/java/core/basesyntax/resources/inputDataForPeriod.csv";
+    private static final String SEPARATE = ",";
+    private static final int INDEX_OPERATION = 0;
+    private static final int INDEX_FRUIT = 1;
+    private static final int INDEX_QUANTITY = 2;
+
+    private Transaction getFruitTransaction(String line) {
```

Comment:
this is more of a `parseTransaction()` method, and this is not the responsibility of the file reader service
Create a separate service for this functionality